node/pod.go

```diff
@@ -374,7 +379,10 @@ func (pc *PodController) deletePodHandler(ctx context.Context, key string) (retE
 		span.SetStatus(err)
 		return err
 	}
-
+	if string(k8sPod.UID) != uid {
+		log.G(ctx).Errorf("Pod uid: %v, desired uid: %v", k8sPod.UID, uid)
```

Comment:
Please use withField

Reply:
Also, make this error message "make more sense" - something like:

log.G(ctx).WithField("k8sPodUID", k8sPod.UID).WithField("uid", uid).Warn("Not deleting pod because remote pod has different UID")

Reply:
done